src/main/java/httpserver/HttpRequest.java

```diff
@@ -2,25 +2,35 @@
 
 public class HttpRequest{
     private String resource = "";
+    private String body = "";
 
     HttpRequest(HttpRequestBuilder builder){
          this.resource = builder.route;
+         this.body = builder.body;
      }
 
      public String getResource(){
         return resource;
      }
 
+     public String getBody(){ return body; }
+
 
     public static class HttpRequestBuilder {
 
         private String route = "";
+        private String body = "";
```

Comment:
Should default to the empty string?

Reply:
I think body is fine defaulting to an empty string since it's not always defined. I changed `route` to default to home.